GooglePlayInstant/Editor/AppBundlePublisher.cs

```diff
@@ -66,14 +73,24 @@ public static bool Build(string aabFilePath)
         {
             bool buildResult;
             Debug.LogFormat("Building app bundle: {0}", aabFilePath);
-#if UNITY_2018_4_OR_NEWER
+            // As of February 2019, every released version of Unity natively supporting AAB has used Android Gradle
+            // Plugin 3.2.0 which includes bundletool 0.5.0. Bundletool 0.6.0+ is needed for uncompressNativeLibraries
+            // and version 0.6.1+ is needed for uncompressNativeLibraries with instant apps.
+            // One can #define PLAY_INSTANT_ENABLE_NATIVE_ANDROID_APP_BUNDLE to build using the native AAB builder.
+#if PLAY_INSTANT_ENABLE_NATIVE_ANDROID_APP_BUNDLE
```

Comment:
I like that we're giving developers this option.

Reply:
Ack. I needed to change this anyway since the 2019.1 beta is still using plugin 3.2.0, but it should also make it easier for us to test.